moveit_core/collision_detection_fcl/src/collision_env_fcl.cpp

```diff
@@ -309,6 +309,45 @@ void CollisionEnvFCL::checkRobotCollisionHelper(const CollisionRequest& req, Col
   }
 }
 
+bool CollisionEnvFCL::checkCollisionBetweenObjectGroups(std::vector<std::string> const& object_group1,
+                                                        std::vector<std::string> const& object_group2) const
```

Comment:
I would expect the possibility to specify only *one* set of objects and check against everything else.

Reply:
That sounds like a good idea. I will look into that as well.